backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -602,7 +602,7 @@ SearchException invalidFieldValueType(@FormatWith(ClassFormatter.class) Class<?>
 	@Message(id = ID_OFFSET + 113,
 			value = "Invalid cardinality for projection on field '%1$s': the projection is single-valued,"
 					+ " but this field is multi-valued."
-					+ " Make sure to call '.multi()' when you create the projection.")
+					+ " Make sure to call '.accumulator(...)' when you create the projection.")
```

Comment:
This makes me wonder about the name... While `ProjectionAccumulator` was fine(-ish) for an SPI that most likely nobody except us will use, it's arguably a bit weird and I don't think people will look for that word when looking for this feature.

I suppose `ProjectionCollector` is a better name, but then people might wonder why we didn't just use `java.util.stream.Collector`. From the javadoc it's mainly because of unnecessary features in `Collector` and because of the `transform` stuff.